Software/Python/easygopigo3.py

```diff
@@ -2352,10 +2355,10 @@ def read_raw_sensors(self):
         """
```

Comment:
Could you add the following line in the documentation of the `read_raw_sensors` method right after the `:rtype: list[int]` thing?
```
:raises IOError: If the line follower is not responding. 
```

Reply:
Done!